highlevel/robot/controller/strategy.py

```diff
@@ -47,21 +50,71 @@
     (Vector2(300, 1450), True),
 ]
 
+PATH_MIRRORED = [(Vector2(pos.x, 2000 - pos.y), reverse)
+                 for pos, reverse in PATH]
+
+
+class ControlMode(Enum):
```

Comment:
Not sure what it has to do with the "strategy" of the robot?

same thing for `set_mode`

When opening strategy.py I would expect to read what is the very high level algorithm of the robot, not hardware/low-level specific code